packages/block-editor/src/utils/use-last-focus.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * WordPress dependencies
+ */
+import { createRef } from '@wordpress/element';
+
+/**
+ * Returns the element of the last element that had focus when focus left the editor canvas.
+ *
+ * @return {Object} Object with a lastFocus ref and setLastFocus.
+ */
+const lastFocus = createRef();
```

Comment:
Unfortunately, that is not a good approach because this is a global singleton, meaning that if you have multiple block editors in the same page, you'll end up with conflicts (they'll be sharing the same ref)

Reply:
Ah, I didn't consider multiple block editors on one page. Do you have any recommended routes to explore? I'll look at it again.

Reply:
I'm not familiar enough with what we're trying to achieve here but a private selector/action could work without changing any logic.